modules/fiftyonedegrees/devicedetection/request_headers_extractor_test.go

```diff
@@ -0,0 +1,118 @@
+package devicedetection
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/51Degrees/device-detection-go/v4/dd"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractEvidenceStrings(t *testing.T) {
```

Comment:
Nitpick: for all test cases here, replace spaces in `name` field with underscores which is what `t.Run` will do when it encounters spaces (e.g. `"Ignored query evidence"` --> `"Ignored_query_evidence"`). This will make it easy to find a failing test via copy and search.